gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -184,7 +275,14 @@ def run(
                 w_i2Ui1_inlier_measurements.append(w_i2Ui1)
 
         # Run the optimizer
-        wti_values = TranslationRecovery(w_i2Ui1_inlier_measurements).run(scale_factor)
+        algorithm = TranslationRecovery()
+        w_i2ti1_priors = self.__get_priors_hilti_rig(i2Ti1_priors, wRi_list)
```

Comment:
We should ensure this is only called for Hilti datasets.

Reply:
Maybe create a new class like how we did for Hilti?